contrib/scripts/checkpr.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if ! which curl >/dev/null || ! which jq >/dev/null; then
+    echo "This tool requires 'curl' and 'jq' utilities to work correctly. Please install them."
+    exit 1
+fi
+
+if [ $# -eq 0  ]
+then
+	echo "Usage: checkpr.sh PULL_REQUEST_ID"
+	exit
+fi
+
+PR_ID=$1
+
+result="null"
+
+until [ "$result" != "null" ]; do
+	sleep 60
+	result=$(curl -s https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-${PR_ID}/lastBuild/api/json | jq ".result")
```

Comment:
Will it still works with the new ginkgo jenkins jobs?

Reply:
Probably not since the job name will be different. That URL can easily be updated later. Should not block this.